File: lib/libsignal-service/src/main/java/org/whispersystems/signalservice/api/payments/CurrencyConversions.java
package org.whispersystems.signalservice.api.payments;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public final class CurrencyConversions {
  @JsonProperty
  private List<CurrencyConversion> currencies;

  @JsonProperty
  private long timestamp;

  public List<CurrencyConversion> getCurrencies() {
    return currencies;
  }

  public long getTimestamp() {
    return timestamp;
  }
}

File: lib/libsignal-service/src/main/java/org/whispersystems/signalservice/api/payments/CurrencyConversions.kt
/*
 * Copyright 2025 Signal Messenger, LLC
 * SPDX-License-Identifier: AGPL-3.0-only
 */

package org.whispersystems.signalservice.api.payments

data class CurrencyConversions(
  val currencies: List<CurrencyConversion> = emptyList(),
  val timestamp: Long = 0
)
